Search Update

MadKea opened the pull request:
Added UX features such as a toggle to a different perspective of a listing. This card allows one to zoom into a specific listing area. also fixed the map in place so it is always in view even with a lot of listings. Changed the filters to add the toggle. So, Murray, can you help keep things as uniform. I just used the old design. After the merge I will update to include area info stats and recommendation stats.

MadKea committed:
All tab added (sell + rent + all)

MadKea committed:
Update

MadKea committed:
Update

did not fix map into place as is not on dev

MadKea committed:
Update

MadKea committed:
Seach cards done

Just need to add area listing values. Otherwise, review and let's merge

MadKea committed:
Merge branch 'DEVELOPMENT_v2' into cardUpdate

MadKea committed:
Update search.page.html

Rob-0ff committed:
Added the listing area scores to the listing cards

Rob-0ff reviewed (changes requested):
I think it would be best to keep the old search bar, the cards look great so those must definitely stay

MadKea committed:
Updated search bar back to OG

Managed to include the toggle

MadKea committed:
mylisting revamp start

Adding UX feature to the design of mylistings and saved listings

MadKea committed:
My listings done.

Basically done. Just needs to be reviewed and I want to add more UX stuffies.

MadKea committed:
savedlistings shortcut added

Done. Though, all the cards might need edit after the description update has been merged.

MadKea committed:
SavedListings done

I could add a button to swap cards just like search. is it too much?

MadKea committed:
Ended up added both cards

MadKea committed:
Saved + My listings done

Outstanding: might change description after merge.
Waiting for recSys to use colour for area scores

MadKea committed:
Merge branch 'DEVELOPMENT_v2' into cardUpdate

Rob-0ff committed:
Merge branch 'DEVELOPMENT_v2' into cardUpdate

MadKea committed:
Fixed images (icons)

Review in need

Thapelo64 committed:
removal of "All"
